src/api.js

```diff
@@ -0,0 +1,43 @@
+import { gql } from '@apollo/client';
+  
+const GET_ALL_ITEMS = gql`
+query getItems {
+  queryItems {
+      id
+      name
+      productType
+      image
+      description
+      color
+      quantity
+      size
+  }
+}`
+
+  // an example of a post
```

Comment:
I assume you want to keep all the commented out 'example of a post' in for later use